app/api/Spaces.scala

```diff
@@ -167,9 +177,17 @@ class Spaces @Inject()(spaces: SpaceService,
     toJson(Map("id" -> space.id.stringify,
       "name" -> space.name,
       "description" -> space.description,
+      "creator" -> space.creator.stringify,
       "created" -> space.created.toString))
   }
 
+  def userToJson(user: User) = {
+    toJson(Map("id" -> user.id.stringify,
+      "email" -> user.email.get
+    ))
+  }
```

Comment:
I don't know scala well enough, but since you are returning the object of user, why don't you cast every field available?

Reply:
We already create the user in v2 and have the id from v1, so this is enough for migrating.